### PR TITLE
ITSASU-677

### PR DESCRIPTION
**Pull request by pradeepkjobs**

### CODE REVIEW

File: app/views/agent/eligibility/AreYouASoleTrader.scala.html [hunk 1]
```diff
@@ -27,32 +27,47 @@
 @import forms.agent.SoleTraderForm
 @import views.helpers.RadioOption
 @import forms.submapping.YesNoMapping
-@import views.html.helpers.continueButton
+
```
Comment (sb-dev) [inline app/views/agent/eligibility/AreYouASoleTrader.scala.html:30]: Can you please remove unused imports?

File: app/views/agent/eligibility/AreYouASoleTrader.scala.html [hunk 2]
```diff
@@ -27,32 +27,47 @@
...
+implicit appConfig: AppConfig
+)
 
-    @back_link(backUrl)
+@(soleTraderForm: Form[YesNo], postAction: Call, startDateLimit: String, backUrl: String)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+@mainTemplate(title = messages("agent.eligibility.sole_trader.heading", startDateLimit), optForm = Some(soleTraderForm), backLink = Some(backUrl)) {
 
-    @summaryErrorHelper(soleTraderForm)
+    <h1 class="govuk-heading-l">@messages("agent.eligibility.sole_trader.heading", startDateLimit)</h1>
+    <p class="govuk-body">@messages("agent.eligibility.sole_trader.line")</p>
```
Comment (sb-dev) [inline app/views/agent/eligibility/AreYouASoleTrader.scala.html:51]: The following line is not needed, it was not part of the old template:
```
<p class="govuk-body">@messages("agent.eligibility.sole_trader.line")</p>
```

File: app/views/agent/eligibility/AreYouASoleTrader.scala.html (hunk 1 above)
Comment (pradeepkjobs) [reply app/views/agent/eligibility/AreYouASoleTrader.scala.html:30]: Updated as per comments  above

File: app/views/agent/eligibility/AreYouASoleTrader.scala.html (hunk 2 above)
Comment (pradeepkjobs) [reply app/views/agent/eligibility/AreYouASoleTrader.scala.html:51]: removed  the unused  code

**Review by sb-dev — APPROVED** [review]

**Pull request merged**